keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -25,23 +25,28 @@ type KubeCloudInstTool struct {
 func (cu *KubeCloudInstTool) InstallTools() error {
 	cu.SetOSInterface(GetOSInterface())
 	cu.SetKubeEdgeVersion(cu.ToolVersion)
+	fmt.Println("beforeinstallkubeedge")
 
 	err := cu.InstallKubeEdge()
 	if err != nil {
 		return err
 	}
 
 	err = cu.generateCertificates()
+	fmt.Println("generateCertificates")
 	if err != nil {
 		return err
 	}
 
 	err = cu.tarCertificates()
+	fmt.Println("tarCertificates")
 	if err != nil {
 		return err
 	}
+	time.Sleep(5 * time.Second)
```

Comment:
Is this sleep very much necessary? If not, please remove, or else check if you can reduce it to 1 second.